rapids-cmake/test/install_relocatable.cmake

```diff
@@ -0,0 +1,112 @@
+#=============================================================================
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_test_install_relocatable
+-------------------------------
+
+.. versionadded:: v23.04.00
+
+Install the needed `ctest` infrastructure to allow installed tests to be run
+by `ctest` in parallel with GPU awarness.
+
+  .. code-block:: cmake
+
+    rapids_test_install_relocatable(INSTALL_COMPONENT_SET <component>
+                                    DESTINATION <relative_path>
+                                    [EXCLUDE_FROM_ALL])
+
+Add ...
```

Comment:
What does this line mean? Is this an incomplete sentence?

Reply:
Looks like I forgot to write all the documentation for what the function does. I will go ahead and actually write the document